compose.yaml

```diff
@@ -0,0 +1,84 @@
+name: wevote
+
+services:
+  db:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.db
+      tags:
+        - "${COMPOSE_PROJECT_NAME}-db:latest"
+    container_name: "${COMPOSE_PROJECT_NAME}-db"
+    environment:
+      POSTGRES_DB: WeVoteServerDB
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-secret}
+    env_file: .env
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    networks:
+      - ${COMPOSE_PROJECT_NAME}
+    ports:
+      - "5000:5432"
```

Comment:
Do we need all these ports?

Reply:
You're right that we just need port 5432.